cup_of_join_exercise.py

```diff
@@ -0,0 +1,17 @@
+def join(*args, sep="-"):
+    if not args:
+        return []
+    elif len(args) == 1:
+        return args[0]
+
+    amount_of_args = len(args)
+    new_list = []
+
+    for i in range(0, amount_of_args - 1):
+        new_list += args[i]
+        new_list.append(sep)
+
+    return new_list + args[amount_of_args - 1]
+
+
+print(join([1]))
```

Comment:
Use _name==main_ - 
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/

Reply:
fixed it